### ISSUE TITLE
Shortcuts to Apps installed by Microsoft Store

### ISSUE THREAD

**Issue by hitechpanchal** [tjackenpacken/taskbar-groups#296]
Apps from Microsoft Store, do not appear in the list. I tried creating a shortcut from shell:AppsFolder, but then it seems, due to the folder restrictions, that does not works.

**Comment by Adam-Dalloul** [comment]
Experiencing the same issue here.

**Comment by willowilly** [comment]
Any fix for this issue?

**Comment by Riversiderepeat** [comment]
This is not an issue with TBG. It's Microsoft, yet again, dumbing down stuff so power users can't figure out what the heck they did. There is a workaround for this problem [here](https://answers.microsoft.com/en-us/windows/forum/all/solved-can-you-create-your-own-shortcuts-to-metro/00549336-f6bc-408f-b0e5-c7c694fbb35a).

You're welcome!

**Comment by Runel2** [comment]
The easiest way to fix this is to set yourself as owner:
-Right click on "WindowsApps" folder -> Properties -> Security -> Advenced
Most likely you'll see "Owner: TrustedInstaller" and Modify right next to it, click on Modify
And type the session's username, then click on "verify name" and "ok"
- then you select "apply", and now you can access the insides of the folder, and create a shortcut

You're welcome :)

**Comment by Riversiderepeat** [comment]
@Runel2 

Well thanks for the effort but this doesn't solve the issue at hand. I AM the owner of said folder and that doesn't allow for actually finding the files that need to be launched. The link to how to do that is found in my previous post.

**Comment by KV-Mark** [comment]
So i found a solution:
- Press the Windows Key + R to open the Run dialog box.
- Type shell:AppsFolder and click Ok. This will open the Applications folder.
- In the Applications folder, find the app for which you want to create a shortcut.
- Right-click on the app icon and select the Create shortcut option.
- In the warning prompt, click Yes.

this will add the shortcut to the desktop, pop them into a folder and u will be able to add them from there to the taskbar group

**Comment by Riversiderepeat** [comment]
@KV-Mark Yes. Those are the exact same instructions I linked to above. I suppose it's no harm in having them here as well.
